src/modules/feature-modules/innovator/pages/innovation/how-to-proceed/recommend-need-reassessment/recommend-needs-reassessment.ts

```diff
@@ -0,0 +1,35 @@
+import { Component } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { CoreComponent } from '@app/base';
+
+@Component({
+  selector: 'app-innovator-pages-recommend-needs-reassessment',
+  templateUrl: './recommend-needs-reassessment.html'
+})
+export class PageInnovationRecommendNeedsReassessment extends CoreComponent {
+  innovationId: string;
+  baseUrl: string;
+
+  constructor(private activatedRoute: ActivatedRoute) {
+    super();
+
+    this.innovationId = this.activatedRoute.snapshot.params.innovationId;
+
+    this.baseUrl = `/innovator/innovations/${this.innovationId}`;
+
+    this.setPageTitle('We recommend you submit for needs reassessment', { width: '2.thirds', size: 'l' });
+    this.setBackLink('Go back', `${this.baseUrl}/how-to-proceed`);
```

Comment:
Here, should you pass a query param to select the previously chosen option when going back to the how to proceed page?